choose_disk.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Defines deployment design and assigns role to server groups
```

Comment:
include here = in the repository or in this directory ?

I think it's pretty useful to get sample code to see how to use it